src/ott/problems/quadratic/quadratic_problem.py

```diff
@@ -383,19 +383,19 @@ def convert(
       return self
 
     (geom_xx, geom_yy, geom_xy, *children), aux_data = self.tree_flatten()
-    (s1, s2, s3) = jax.random.split(jax.random.PRNGKey(seed), 3)[:, 0]
+    (k1, k2, k3) = jax.random.split(rng, 3)
```

Comment:
```suggestion
    rng1, rng2, rng3 = jax.random.split(rng, 3)
```
(No parenthesis to be consistent, and k -> rng